src/components/JSONOutput.js

```diff
@@ -235,18 +255,19 @@ export default function JSONOutput(props) {
   };
 
   const renderFileTreeView = () => {
-    const order = ['StructureDefinitions', 'ValueSets', 'CodeSystems', 'Instances'];
+    const order = ['StructureDefinitions', 'ValueSets', 'CodeSystems', 'Instances', 'Unknown Type'];
     const grouped = groupBy(fhirDefinitions, (val) => {
       if (['StructureDefinition', 'ValueSet', 'CodeSystem'].includes(val.resourceType)) return `${val.resourceType}s`;
-      return 'Instances';
+      if (val.resourceType != null) return 'Instances';
+      return 'Unknown Type';
```

Comment:
Ok so I used my unique talents to make this crash here. I can't figure out how though. The error happened because `val` ended up undefined somehow. Can you see how that would happen? Maybe just to be safe you could return `Unkown Type` if `val` is undefined?

I'm sorry I can't give more info, but I'm really not sure how to replicate it regularly.

Reply:
Hm yeah I'm struggling to see how/where that could be coming from. To be safe, I'll add in `val?.resourceType` in these two checks so that it will fall to the `Unknown Type` case if it's null. There are also a few other places to add `?.` to make it safe if a `fhirDefinition` is undefined.

Reply:
This should be addressed in 23a750c. I'm still not sure what the original root cause was, but hopefully the addresses the symptoms.